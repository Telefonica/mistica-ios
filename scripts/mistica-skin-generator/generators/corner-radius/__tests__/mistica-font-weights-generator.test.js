import fs from 'fs';
import {resolve} from 'path';
import {generateMisticaCornerRadius} from '../mistica-corner-radius-generator';

test('generateMisticaCornerRadius', () => {
    const anyBrandTokensPath = resolve(__dirname, 'any-brand-tokens.json');
    const anyBrandTokens = JSON.parse(fs.readFileSync(anyBrandTokensPath, 'utf-8'));
    const result = generateMisticaCornerRadius(anyBrandTokens);

    expect(result).toMatchSnapshot();
});
