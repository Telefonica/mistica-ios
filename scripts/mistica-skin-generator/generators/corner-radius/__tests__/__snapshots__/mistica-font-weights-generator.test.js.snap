// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`generateMisticaCornerRadius 1`] = `
"
// Generated using Make
// DO NOT EDIT

import Foundation

public enum MisticaRadiusConstants {
    static let roundedRadius: CGFloat = 999.0
}

public protocol MisticaCornerRadius {
    var avatar: CGFloat { get }
    var bar: CGFloat { get }
    var button: CGFloat { get }
    var checkbox: CGFloat { get }
    var container: CGFloat { get }
    var indicator: CGFloat { get }
    var input: CGFloat { get }
    var legacyDisplay: CGFloat { get }
    var popup: CGFloat { get }
    var sheet: CGFloat { get }
    var mediaSmall: CGFloat { get }
}
"
`;
